rpm/libblockdev.spec

```diff
@@ -10,45 +10,15 @@ BuildRequires: pkgconfig(glib-2.0)
 BuildRequires: pkgconfig(gobject-introspection-1.0)
 BuildRequires: pkgconfig(udev)
 BuildRequires: pkgconfig(libkmod)
+BuildRequires: pkgconfig(libkeyutils)
 BuildRequires: gettext-devel
 BuildRequires: autoconf
 BuildRequires: autoconf-archive
 BuildRequires: automake
 BuildRequires: libtool
 BuildRequires: libfdisk-devel
-# i=1; for j in 00*patch; do printf "Patch%04d: %s\n" $i $j; i=$((i+1));done
```

Comment:
Please don't remove the comment which includes a command to generate the list of patches for `Patch:`.

Reply:
I don't think it makes sense anymore with just one patch. Nobody is going to need much help in doing that. I'm hopeful that we can stay with such a low amount of them and even get rid of the remaining one eventually.